src/components/ManaCalculator/components/OtherParametersSection.tsx

```diff
@@ -36,13 +36,17 @@ export function OtherParametersSection() {
       <label className='inlined-label'>Initial epoch</label>
       <input
         className='mana_calculator__compact inlined'
+        type='number'
+        min='0'
```

Comment:
Should be one afair. @begonaalvarezd ?
```suggestion
        min='1'
```

Reply:
@VmMad this is `1` indeed